components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilder.java

```diff
@@ -648,5 +652,28 @@ private boolean isValidIdToken(JWTClaimsSet jwtClaimsSet) {
         return isValidIdToken;
     }
 
+    private ArrayList<String> getEssentialClaimList(String essentialClaims){
```

Comment:
As we use the same method when developing the user info response shall we move the logic to a util class, so we can reuse the same method.